src/modules/common/templates/dropdown.tsx

```diff
@@ -31,12 +32,7 @@ const Dropdown: React.FC<DropdownProps> = ({ options, label, className, children
       >
         {label}
         {children}
-        <ChevronRightIcon
-          className={tw(
-            "text-light-text dark:text-dark-text size-4 transition-transform duration-200",
-            { "rotate-90": isOpen }
-          )}
-        />
+        <Icon icon={ChevronRightIcon} className="w-4 h-4" />
```

Comment:
You forgot to put back the classes that already had the icon before.

Remember that you have to pass a created object, however, you are passing an unbuilt object, use `< />`.

```suggestion
        <Icon
          icon={<ChevronRightIcon />}
          className={tw(
            "transition-transform duration-200",
            { "rotate-90": isOpen }
          )}
        />
```